Realm/RLMArray.h

```diff
@@ -349,14 +348,14 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark - Unavailable Methods
 
 /**
- -[RLMArray init] is not available because RLMArrays cannot be created directly.
- RLMArray properties on RLMObjects are lazily created when accessed, or can be obtained by querying a Realm.
+ `- (RLMArray)init` is not available because `RLMArray`s cannot be created directly.
```

Comment:
Why change from `-[RLMArray init]` to `- (RLMArray)init`? The former is the conventional way of referring to an instance method.